packages/simcore-sdk/tests/unit/test_storage_client.py

```diff
@@ -53,32 +54,46 @@ async def test_get_storage_locations(
     assert result[0].id == 0
 
 
-async def test_get_download_file_presigned_link(
+@pytest.mark.parametrize(
+    "link_type, expected_scheme",
+    [(LinkType.PRESIGNED, ("http", "https")), (LinkType.S3, ("s3", "s3a"))],
```

Comment:
just found out the difference between `s3a`, `s3n` and `s3` https://stackoverflow.com/a/33356421/2855718

Reply:
thx